collaborative_filtering/evaluation/accurancy.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+import math
+import textwrap
+
```

Comment:
I would rename this file and all instances of "accurancy" to "accuracy". It seems to be just a typo.